#!/usr/bin/env python
PACKAGE = "offboard_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pid_kd", double_t, 0, "An Integer parameter", -0.75, 0.09, 0.75)
gen.add("pid_ki", double_t, 0, "A double parameter",   -0.75, 0.00005, 0.75)
gen.add("init_pos_x", double_t, 0, "X position", -20.0, 1, 20.0)
gen.add("init_pos_y", double_t, 0, "Y position", -20.0, 1, 20.0)
gen.add("init_pos_z", double_t, 0, "Z position", 2.0, 1, 20.0)

exit(gen.generate(PACKAGE, "offboard_control", "pidController"))
